pkg/settings/settings.go

```diff
@@ -24,6 +26,7 @@ var (
 	ServerVersion          = NewSetting("server-version", "dev")
 	UIIndex                = NewSetting("ui-index", "https://releases.rancher.com/harvester-ui/latest/index.html")
 	UIPath                 = NewSetting("ui-path", "/usr/share/rancher/harvester")
+	UIOfflinePreferred     = NewSetting("ui-offline-preferred", "dynamic")
```

Comment:
refer to https://github.com/rancher/harvester/blob/8295da51c1e6a7461441083d3ce63ed841211f20/pkg/server/ui/api.go#L13-L35
and instead of using the `local` config in the ui-index setting, the offline preference is now checked with https://github.com/rancher/harvester/blob/8295da51c1e6a7461441083d3ce63ed841211f20/pkg/settings/settings.go#L126-L128

Reply:
Suggested change:
```
APIUISource     = NewSetting("api-ui-source", "auto") // Options are 'auto', 'external' or 'bundled'
```

1. To differentiate API-UI and the UI. 
2. The name might be confusing if it's a valid setting for both online and offline environments.
3. Have comments for available options so that people don't need to check the implementation details.

Reply:
@gitlawr updated thanks